pkg/mediationcontainer/client_protobuf_endpoint.go

```diff
@@ -109,14 +109,11 @@ func (endpoint *ClientProtobufEndpoint) waitForServerMessage() {
 				glog.V(4).Infof(logPrefix+" closed MessageChannel %+v", endpoint.ParsedMessageChannel)
 				return
 			//default:
-			case rawBytes, ok := <-endpoint.transport.RawMessageReceiver(): // block till  the message bytes from the transport channel,
+			case rawBytes := <-endpoint.transport.RawMessageReceiver(): // block till  the message bytes from the transport channel,
```

Comment:
Does this change mean the sender has been changed to never close the channel, unlike it used to?